trillian/ctfe/handlers.go

```diff
@@ -219,6 +219,25 @@ type CertValidationOpts struct {
 	extKeyUsages []x509.ExtKeyUsage
 }
 
+// NewCertValidationOpts can be used for struct creation outside of ctfe package.
```

Comment:
(nit) Comment describes how function is used, not what it does.  Maybe just "NewCertValidationOpts builds validation options based on parameters"?

Reply:
Done.